build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eu
+
+readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+readonly THIS_SCRIPT_PATH="$(dirname "$THIS_SCRIPT")"
+
+cd "$THIS_SCRIPT_PATH"
+
+build_file_list() {
```

Comment:
Can we add a few more maintenance comments?

Reply:
Sure thing